Formula/yara.rb

```diff
@@ -9,7 +9,7 @@ class Yara < Formula
 
     # fixes a variable redefinition error with clang (fixed in HEAD)
     patch do
-      url "https://github.com/plusvic/yara/pull/261.diff"
```

Comment:
We could use Baptiste's PR for simplicity, but probably should note the commit it was actually fixed in as well.

The fix commit was https://github.com/VirusTotal/yara/commit/11fd5e4784fa135cbdb64906cc628edb9e9d50ee, and Baptiste's PR was https://github.com/VirusTotal/yara/pull/261.diff